src/protocol/utils.ts

```diff
@@ -81,7 +81,7 @@ export const DisallowEverythingProxyHandler: ProxyHandler<object> = {
   //  return false;
   //},
   get() {
-    NotAllowed();
```

Comment:
Aha I'm often mystified by that "NotAllowed" error.

Reply:
It's just so helpful. I'll miss it.